### PR TITLE
Clarify "executable" property in gradle.properties

### PR DESCRIPTION
**Pull request by vieiro**
Let's improve the importance of the`executable=false` or `executable=true` setting in `gradle.properties`, so people can see it directly in README.md

### CODE REVIEW

File: app/src/main/resources/common/README.md.mustache [hunk 1]
```diff
@@ -91,14 +91,28 @@ On a successful deployment via the following methods, the server will be availab
 {{#configServer}}* `https:/localhost:8888/casconfigserver`{{/configServer}}
 {{#discoveryServer}}* `https://localhost:8761`{{/discoveryServer}}
   
-## Executable WAR
+## Build and run CAS with an embedded server
 
-Run the server web application as an executable WAR. Note that running an executable WAR requires CAS to use an embedded container such as Apache Tomcat, Jetty, etc.
+Run the server web application as an executable WAR.
```
Comment (mmoayyed) [inline app/src/main/resources/common/README.md.mustache:96]: This is misleading, I think. The title for this section says: "build and run CAS with an embedded server" and you're describing executable WARs. These have nothing to do with each other. If tou want to describe an executable WAR deployment, you should start a separate section, and not modify what exists already.

File: app/src/main/resources/common/README.md.mustache [hunk 2]
```diff
@@ -91,14 +91,28 @@ On a successful deployment via the following methods, the server will be availab
...
+
+Note that running an executable WAR requires CAS to use an embedded container such as Apache Tomcat, Jetty, etc.
 
 {{#appServer}}The current servlet container is specified as `{{appServer}}`.{{/appServer}}
 {{^appServer}}No servlet container is specified for the current build. Examine your `gradle.properties` file
 and modify the `appServer` property to point to the appropriate container of choice.{{/appServer}}
 
+### Building
+
+In order to build an executable war, update `gradle.properties` and set `executable=false`, then run:
```
Comment (mmoayyed) [inline app/src/main/resources/common/README.md.mustache:106]: This is not maintainable; manual instructions like this in the README are not going to be looked after by maintainers. Explanations like this should go into the gradle.properties file where the change actually belongs.

File: app/src/main/resources/common/README.md.mustache [hunk 3]
```diff
@@ -109,7 +123,9 @@ Or via:
 ./gradlew[.bat] run
 ```
 
-Debug the CAS web application as an executable WAR:
+### Debugging
+
+To debug the CAS web application as an executable WAR:
```
Comment (mmoayyed) [inline app/src/main/resources/common/README.md.mustache:128]: This has nothing to do with executable WARs.

File: app/src/main/resources/common/README.md.mustache [hunk 4]
```diff
@@ -121,12 +137,36 @@ Or via:
 java -Xdebug -Xrunjdwp:transport=dt_socket,address=5000,server=y,suspend=y -jar build/libs/{{appName}}.war
 ```
 
-Run the CAS web application as a *standalone* executable WAR:
+## Build and run CAS as a *standalone* executable WAR
+
+This allows you to run the CAS web application as a *standalone* executable WAR on Linux and Mac.
+
+The build system will add a startup script inside the war, that makes it executable.
```
Comment (mmoayyed) [inline app/src/main/resources/common/README.md.mustache:144]: This is not something we can add; it's not maintainable if it ever changes since the behavior is controlled by an external library and they could change this at any moment.

File: app/src/main/resources/common/README.md.mustache [hunk 5]
```diff
@@ -121,12 +137,36 @@ Or via:
...
 
+### Running
+
+Once built, the generated `build/libs/cas.war` will be directly executable on Linux and Mac:
+
+```bash
+./build/libs/cas.war
+```
+
+You can take a look at the startup script inside the war by running the command:
```
Comment (mmoayyed) [inline app/src/main/resources/common/README.md.mustache:162]: This bit is also not maintainable. This behavior could change at any moment by the external library that handles this sort of thing, and we can manage the change automatically, or per CAS version in the docs like this. Explanations like this belong in a blog.

**Review by mmoayyed — CHANGES_REQUESTED** [review]

**Comment by vieiro** [comment]
Thanks for your review, @mmoayyed . 
Let me rephrase things and try again whenever I have some time to waste on this.
